source/angular-date-picker.js

```diff
@@ -6,43 +6,55 @@ angular.module( "vokal.datePicker", [] )
     {
         "use strict";
 
+        var defaultFormat = "M/d/yyyy";
+
+        function validateDate( date )
+        {
+            return date && angular.isFunction( date.getTime ) && !isNaN( date.getTime() );
```

Comment:
Seems like strictly speaking, this should always return a boolean, so you might want to change `date` to `!!date`